src/einsteinpy/symbolic/vaccum_metrics.py

```diff
@@ -0,0 +1,26 @@
+import sympy
+
+from .metric import MetricTensor
+
+
+def SchwarzschildMetric(symbolstr="t r theta phi"):
+    """
+    Returns  Metric Tensor  of symbols of Schwarzschild Metric.
+    Parameters
```

Comment:
A newline is needed before `Parameters`. Maybe, you can see from above codebase, how to write docstrings elegantly.